packages/utils/typescript/lib/generators/content-types/index.js

```diff
@@ -5,6 +5,11 @@ const { factory } = require('typescript');
 const { models } = require('../common');
 const { emitDefinitions, format, generateSharedExtensionDefinition } = require('../utils');
 
+const NO_CONTENT_TYPE_PLACEHOLDER_COMMENT = `/*
+ * The app doesn't have any content-type yet.
```

Comment:
```suggestion
 * The app doesn't have any content-types yet.
```

Reply:
Is it even possible to not have any content types though, since we combine it with our internal models?

Reply:
Shouldn't be possible in the context of a normal application, but better be on the safe side